website/cue/reference/components/sinks/pulsar.cue

```diff
@@ -16,7 +16,12 @@ components: sinks: pulsar: {
 		acknowledgements: true
 		healthcheck: enabled: true
 		send: {
-			compression: enabled: false
+			compression: {
+				enabled: true
+				default: "none"
+				algorithms: ["none", "lz4", "zlib", "zstd", "snappy"]
```

Comment:
This is failing to validate with Cue because `zlib` is not one of the [options listed in `website/cue/reference.cue`](https://github.com/vectordotdev/vector/blob/815f0b9b99b6dffcbb61b09d0a44f1c124cbd809/website/cue/reference.cue#L24). It needs to be added there as well.

Reply:
😭 one day we'll be free from cue

Reply:
Fixed